### PR TITLE
Add utm package to doc.

### LINKED ISSUE #1812: A UTM converter?
This is merely for discussion. I've been dealing with small dense array these days. It helps with coding (and is more intuitive) converting la&lo to UTM location. Do we have something built already?

I did found [this page](https://docs.obspy.org/tutorial/code_snippets/coordinate_conversions.html) in the doc that teaches you to use [pyproj](https://pypi.python.org/pypi/pyproj). However, it does not seem to solve my problem. I found this [utm](https://pypi.python.org/pypi/utm) on PyPI and has been using it for a while. It is pretty reliable and fast. I think we should have a page mentioning it in case someone else wants to make the conversion as I do.

### PR DESCRIPTION
**Pull request by lijunzh**
+DCOS

### What does this PR do?
Add reference about UTM location convert to obspy docs.

### Why was it initiated?  Any relevant Issues?
Resolves #1812

### CODE REVIEW

**Comment by trichter** [comment]
+DOCS to build the docs for this PR

**Comment by lijunzh** [comment]
> +DOCS to build the docs for this PR

How should I do that?

**Comment by trichter** [comment]
This "+DOCS"  key word in a comment of a pull request starts automatically the docs build bot (I think after the next commit). It will be shown in the checks panel.

**Comment by trichter** [comment]
What's still needed is some action here to kick-start the docs build. For example, an empty commit or a rebase with a subsequent forced push.

**Comment by Jollyfant** [comment]
Actually I think the docs are here?

docs.obspy.org/pull-requests/1814/

**Comment by lijunzh** [comment]
@trichter @Jollyfant  Thanks a lot. Yes, I can see the docs now. There is still some problem with the docs that I didn't expected. I will try to fix them shortly.

**Comment by lijunzh** [comment]
@trichter @Jollyfant I think it is ready for merge. You can check the current version of docs at

**Comment by trichter** [comment]
The UTM link is broken. A dot is missing in the link. Otherwise, it can be merged from my point of view.

File: misc/docs/source/tutorial/code_snippets/coordinate_conversions.rst [hunk 1]
```diff
@@ -22,3 +22,21 @@ coordinates of two German stations to the regionally used Gauß-Krüger system:
 
 .. _`pyproj`: https://pypi.python.org/pypi/pyproj
 .. _`EPSG`: https://www.epsg-registry.org/
+
+
+Another common usage is to convert location information in latitude and
+longitude to `Universal Transverse Mercator coordinate system (UTM)`_. This is
+especially useful for large dense array in a small area. Such conversion can
```
Comment (Jollyfant) [inline misc/docs/source/tutorial/code_snippets/coordinate_conversions.rst]: Just a few typos.

array => arrays
usages => usage

**Comment by lijunzh** [comment]
@trichter Thanks for pointing that out. My IDE kept breaking lines for me. Must be a miss-delete when removing the newlines.

File: misc/docs/source/tutorial/code_snippets/coordinate_conversions.rst (hunk 1 above)
Comment (lijunzh) [inline misc/docs/source/tutorial/code_snippets/coordinate_conversions.rst]: @ Thanks for the review. Yes, change from "array" to "arrays" makes much sense. English is not my first language. But isn't "usage" an uncountable norm? Anyway, I think the UTM is only a *single* usage of possible location conversion. Thus, I will keep the singular form.

**Comment by Jollyfant** [comment]
Thanks @lijunzh!

**Pull request merged**